jetstream/experimenter.py

```diff
@@ -36,25 +98,101 @@ def _unix_millis_to_datetime(num: Optional[float]) -> Optional[dt.datetime]:
         return dt.datetime.fromtimestamp(num / 1e3, pytz.utc)
 
     @classmethod
-    def from_dict(cls, d) -> "Experiment":
+    def from_dict(cls, d) -> "ExperimentV1":
         converter = cattr.Converter()
         converter.register_structure_hook(
             dt.datetime, lambda num, _: cls._unix_millis_to_datetime(num),
         )
         return converter.structure(d, cls)
 
+    def to_experiment(self) -> "Experiment":
+        """Convert to Experiment."""
+        branches = [Branch(slug=variant.slug, ratio=variant.ratio) for variant in self.variants]
+        control_slug = None
+
+        control_slugs = [variant.slug for variant in self.variants if variant.is_control]
+        if len(control_slugs) == 1:
+            control_slug = control_slugs[0]
+
+        return Experiment(
+            normandy_slug=self.normandy_slug,
+            experimenter_slug=self.slug,
+            type=self.type,
+            status=self.status,
+            active=self.status == "Live",
+            start_date=self.start_date,
+            end_date=self.end_date,
+            proposed_enrollment=self.proposed_enrollment,
+            features=[],
+            branches=branches,
+            reference_branch=control_slug,
+        )
+
+
+@attr.s(auto_attribs=True, kw_only=True, slots=True, frozen=True)
+class ExperimentV4:
+    """Represents a v4 experiment from Experimenter."""
+
+    slug: str  # Normandy slug
+    active: bool
+    features: List[Feature]
+    branches: List[Branch]
+    startDate: dt.datetime
+    endDate: Optional[dt.datetime]
+    proposedEnrollment: int
+    referenceBranch: Optional[str]
```

Comment:
The `v4` experiment object attributes are returned camelCased by the API. The easiest way to handle them was to camelCase the class attributes here too. Maybe there is some other magic we could do instead so that we don't need to diverge from our naming style, but not sure if that is necessary...